### ISSUE TITLE
Fedora: Updating kernels impossible if akmods packages are in use

### ISSUE THREAD

**Issue by Cyborgscode** [linux-surface/linux-surface#997]
Device: Pro 4 
OS: Fedora
Packagesystem: DNF4

As only one Surface Kernel per Time is installed on a system, any additional kernel modul compiled with AKMODS is blocking the update of that kernel package. Usually the oldest unused kernel is deinstalled to make space for the new one, which allows erasing the additional dependencies like akmods. As with only one kernel this is not possible, dnf fails on the dependency check. 

We need to have at least 2 surface kernels installed, to be able to update the kernel correctly as v4l2-loopback is required to make cams available to Firefox & Co. 

=> Suggestion for a solution required <=

**Comment by StollD** [comment]
I tried before to allow multiple installable kernels, but it never worked correctly with our custom specfile. Ideally I would like to reuse the one directly from Fedora, but it seems near impossible to properly change the name of that package.

What we could do is use the specfile from Fedora and bump the epoch value. That would mean our kernel always has priority over the one from the Fedora repo. But it would mean we loose the ability to install them in parallel, which is nice to have to debugging and having a known-good backup in case we mess up.
